frontend/src/Types/interfaces.ts

```diff
@@ -0,0 +1,26 @@
+export interface ISocialPart {
+  _id: string;
+  name: string;
+  image: string;
+  active: boolean;
+}
+export interface ISocial {
+  _id?: string;
+  link: string;
+  social: ISocialPart;
+}
+export interface IUser {
+  _id?: string;
```

Comment:
<img width="1345" alt="Screenshot 2021-04-14 at 15 14 44" src="https://user-images.githubusercontent.com/19636010/114716343-423baf00-9d34-11eb-8ca9-5b055d0b0878.png">

Instead of optionalling all properties, we can use Partial<Type>